### DISCUSSION TITLE
Open3D viewer app is now available for Windows

### DISCUSSION THREAD

**Discussion by pkjaslam** [isl-org/Open3D#6092] [General]
Where i can get the Open3D viewer app for Windows?  or  I need to create a graphical user interface (GUI) to visualize my Lidar Cloud points (las files). Could someone please provide me with Python code that I can use to deploy on my server?

**Comment by ssheorey** [comment]
Hi @pkjaslam see the assets section of the release announcement for all downloads:

https://github.com/isl-org/Open3D/releases

Here is a direct link:

https://github.com/isl-org/Open3D/releases/download/v0.17.0/open3d-app-windows-amd64-v0.17.0.zip
